src/lib/module.ts

```diff
@@ -9,6 +9,15 @@ import { CapabilityExport, AdmissionRequest } from "./types";
 import { setupWatch } from "./watch-processor";
 import { Log } from "../lib";
 
+// This should be the upstream type, needs to be exported from KFC
+export interface Rule {
```

Comment:
Upstream types `PolicyRule`

Reply:
Removed the Rule interface and updated to import and use PolicyRule.